app/namespaces/google_spreadsheet/google_spreadsheet_controller.py

```diff
@@ -0,0 +1,25 @@
+from flask import request
+from flask_restplus import Resource, Namespace
+
+from app.database_etl.tableau_data_connector.google_services_manager import GoogleSheetsManager
+
+google_spreadsheet_ns = Namespace('google_spreadsheet', description='Endpoints for making changes to google sheets')
+
+
+@google_spreadsheet_ns.route("/newsletter", methods=["POST"])
+class AppendNewsletterSignup(Resource):
+    @google_spreadsheet_ns.doc("An endpoint for adding new newslettter signups to the newsletter spreadsheet")
+    def post(self):
+        data = request.get_json()
+        if not data:
+            return {"message": "No input payload provided"}, 400
+
+        sheet_name = "Sheet1"
+        spreadsheet_id = ""
+        range_name = "!A1:B1"
```

Comment:
NP: Will we eventually move this to a constants file or .env? I know these are testing values but might be cleaner to keep them somewhere where we store constants.